packages/abap-deploy-config-writer/src/index.ts

```diff
@@ -60,7 +60,7 @@ async function generate(
 
     // package.json
     // deploy script
-    const deployScript = `${BUILD_SCRIPT} && ${DEPLOY_SCRIPT} --config ${deployConfigFile} && ${RIMRAF} archive.zip`;
+    const deployScript = `${BUILD_SCRIPT} && ${DEPLOY_SCRIPT} --config ${deployConfigFile}`;
```

Comment:
What happens to the `archive.zip` after deployment? Is it agreed that the archive.zip will no longer be removed?

Reply:
Query answered here https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/34368#issuecomment-14257310